.github/FUNDING.yml

```diff
@@ -1,12 +0,0 @@
-# These are supported funding model platforms
```

Comment:
Is this deletion intentional?

Reply:
Yes, this was intentional. The FUNDING.yml as it was was for hugo-academic. We can add the funding file back in if needed, but this is really infrastructure rather than a service we're providing.